Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 MAINTAINER Jon-Pierre Stoermer <jp@dronemapper.com>
 
-EXPOSE 3000

```

Comment:
You could also simply remap the port with docker/docker-compose (e.g. `docker [..] -p 3001:3000`) if you are worried about conflicting ports and two nodes running on the same host!

Reply:
I figured as much, just thought it might be cleaner to let NodeMICMAC live on 3001 .. I'm open to whatever works best

Reply:
Up to you! As a user I would expect things to be on the same port and then use docker to remap it if needed, but if you think it works better by using port 3001, that's ok too.

Reply:
Fair point. I'll do a bit more testing on the WebODM side and see how it works .. starting that integration is why I changed it. https://github.com/dronemapper-io/WebODM/commit/6384f8966f6eead2e0901278802f0257717a0736